README.md

```diff
@@ -112,7 +112,7 @@ public_ssh_keys = [
 
 # The example RISE1 server in OVH, deployed with a Ubuntu image
 # will be configured with 'ubuntu' as the admin user
-install_ssh_user = "ubuntu"
+run_as_user = "ubuntu"
```

Comment:
don't we need to keep install user as this could be different and only used for installing on something like ubuntu? Then for normal operation the user puts their user name? probably that should not even be in this config because as this config is shared with multiple users.

Reply:
Because when we can use different user on different command, not install user for `install`, and `root` for all other commands now.

We can discuss the interactive user prompt is a better solution for this in the meeting.

Reply:
`install_ssh_user` is back now only used for installation.